cmd/helm/load_plugins.go

```diff
@@ -154,7 +154,7 @@ func callPluginExecutable(pluginName string, main string, argv []string, out io.
 func manuallyProcessArgs(args []string) ([]string, []string) {
 	known := []string{}
 	unknown := []string{}
-	kvargs := []string{"--kube-context", "--namespace", "-n", "--kubeconfig", "--kube-apiserver", "--kube-token", "--registry-config", "--repository-cache", "--repository-config"}
+	kvargs := []string{"--kube-context", "--namespace", "-n", "--kubeconfig", "--kube-apiserver", "--kube-token", "--insecure-skip-tls-verify", "--registry-config", "--repository-cache", "--repository-config"}
```

Comment:
flags that interact with the kubeconfig should be prefixed with `--kube-`. Please change this to `--kube-insecure-skip-tls-verify`.

Reply:
Done;)

Reply:
It's still `--insecure-skip-tls-verify` here.